elastic/kibana
# [Stack Monitoring] Remove deprecation warning for Internal Monitoring

## Summary

Closes #111629 

Removes this deprecation warning when self-monitoring is enabled: 
<img width="399" alt="Screen Shot 2021-10-12 at 3 40 51 PM" src="https://user-images.githubusercontent.com/1445834/137175210-ba285998-4d9a-45df-95c7-e0aa206a90f8.png">

To test:
- Disable `monitoring.ui.render_react_app: true` in `kibana.dev.yml`. This warning only shows up in the Angular version.
- Connect Kibana to a new empty cluster
- Open Stack Monitoring and enable self-monitoring instead of Metricbeat monitoring
- Reload the page and ensure no deprecation warning appears
